src/patternfly/components/FormControl/form-control.scss

```diff
@@ -90,6 +90,11 @@ $pf-c-form-control__select--ViewBox: "320" !default;
 
   &:not(textarea) {
     height: var(--pf-c-form-control--Height);
+
+    // truncate overflow text
+    overflow: hidden;
```

Comment:
in this case, it doesn't look like `overflow` or `white-space` are needed in the case of an `input`. I tested on Chrome/FF/Safari, and Firefox on Windows (`text-overflow` doesn't work on Edge).

Reply:
Um, from [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow#Browser_compatibility) Edge 12 supports this. What version of Edge are we supporting?

Reply:
Oh sorry, I don't mean the property itself, I mean just on `input`s. I found this issue - https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/15550368/

And here's a screenshot

![Screen Shot 2019-03-28 at 8 29 59 AM](https://user-images.githubusercontent.com/35148959/55161957-b3e83680-5134-11e9-8205-fbee9031dc00.png)